packages/accounts-password/package.js

```diff
@@ -5,11 +5,12 @@ Package.describe({
   // 2.2.x in the future. The version was also bumped to 2.0.0 temporarily
   // during the Meteor 1.5.1 release process, so versions 2.0.0-beta.2
   // through -beta.5 and -rc.0 have already been published.
-  version: "3.0.3",
+  version: "4.0.0-beta320.1",

```

Comment:
Why does adding argon2 require a major version increase? The change log doesn't mention any breaking changes.

Reply:
The original implementation was breaking, but now we can get away with just a feature version since Argon2 is hidden behind a flag.

Reply:
Yes, the reason was at the beginning it was a version with major breaking changes. Not anymore, so I will respect a minor version bump.